backend/typescript/services/implementations/camperService.ts

```diff
@@ -167,6 +167,77 @@ class CamperService implements ICamperService {
 
     return camperDtos;
   }
+
+  async deleteCampersByChargeId(chargeId: string): Promise<void> {
+    try {
+      const numericalChargeId = Number(chargeId);
+      const campers: Array<Camper> = await MgCamper.find({
+        chargeId: numericalChargeId,
+      });
+
+      if (!campers.length) {
+        throw new Error(`Campers with charge ID ${chargeId} not found.`);
+      }
+
+      const camp: Camp | null = await MgCamp.findById(campers[0].camp);
```

Comment:
just confirming, multiple campers with the same chargeId must all be registered for the same camp?